### ISSUE TITLE
SSEE Launcher not working 

### ISSUE THREAD

**Issue by Jonand931** [elishacloud/Silent-Hill-2-Enhancements#768]
HI! I run the SH2EEsetup.exe file as administrator, and I get this error:

![Captura](https://user-images.githubusercontent.com/129980407/230142857-081e8dbb-9bd1-44ba-81d7-58b4233cbe5c.PNG)

I use Windows 7. I dont know what the problem could be, Thanks.

**Comment by nimbumamu** [comment]
Hello, I'm facing the same issue, even after having Windows 11. Seems like a problem with the server. I tried to look up for solutions but this issue seems to be recent, so I couldn't find any.

**Comment by Polymega** [comment]
If you both grab the latest version of the Setup Tool [from here](https://enhanced.townofsilenthill.com/SH2/install.htm#usingEE), does the issue persist?

**Comment by nimbumamu** [comment]
Yes, it does. Infact I've tried it and retried it in these few days. I've also tried disabling the antiviruses and the windows firewall. The same issue persists.

**Comment by nimbumamu** [comment]
Yes, it does. Infact I've tried it and retried it in these few days. I've also tried disabling the antiviruses and the windows firewall. The same issue persists.

**Comment by Polymega** [comment]
Can you double-check the firewall? Another user had this same issue and the firewall was indeed the cause: https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/710#issuecomment-1465069131

**Comment by nimbumamu** [comment]
I just tried it again, this time, I:
1) allowed my web browser (Chrome) + the setup application through the firewall
2) turned off the default virus protection settings

and there is still no remedy. Can using a different web browser or changing the browser settings cause a possible effect?

**Comment by Jonand931** [comment]
> Can you double-check the firewall? Another user had this same issue and the firewall was indeed the cause: [#710 (comment)](https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/710#issuecomment-1465069131)

I have firewall stopped, antivirus too. That was the first think i did. Doesn't solve the issue.

**Comment by Polymega** [comment]
Can you directly access this file? 
https://files.townofsilenthill.com/SH2EE/_sh2ee_current.csv

**Comment by nimbumamu** [comment]
Oh well, I figured it out! I did the following procedures:
1) turrning off all the 3 firewalls of the system (something I did on prior attempts as well)
2) RESTARTING THE COMPUTER
3) then turing off the virus & threat protection settings of the system
4) USING MICROSOFT EDGE to then download the setup

...and it works! I do not know if using MS Edge is coincidental, but I guess, restarting the system after turing off the firewalls may have been the cause of the fix! 

Also, Thank You so much to you Polymega and your team for the help and support along with the development of this very mod itself!
